src/sections/events/Events.module.scss

```diff
@@ -0,0 +1,119 @@
+@use "src/styles/colors" as colors;
+
+.eventsContainer {
+  margin: 0 3rem;
+  width: auto;
+
+  .header {
+    margin-top: 1.5rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+
+    .headerText {
+      width: fit-content;
+      margin: 0;
+      padding: 0;
+    }
+  }
+
+  .filterContainer {
+    display: flex;
+    flex-direction: row;
+    gap: 0 10px;
+
+    @media only screen and (max-width: 920px) {
```

Comment:
can you make 920px a variable for this file since its reused once here + also because i want it to be clear what the breakpoint is here